config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  mount PasskeysRails::Engine => "/passkeys"
```

Comment:
```suggestion
  mount PasskeysRails::Engine => '/passkeys'

```

Leave a blank line before the following comment, and prefer single quotes rather than double.